__tests__/app.test.js

```diff
@@ -231,6 +231,45 @@ describe("PATCH /api/articles/:article_id", () => {
   });
 });
 
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: should delete comment", () => {
+    return request(app).delete("/api/comments/2").expect(204);
+  });
+  test("404: not found", () => {
```

Comment:
I would be more descriptive with this test description - what status code and message are you expecting and why?